functions.php

```diff
@@ -333,9 +333,20 @@ function independent_publisher_comment_count( $count ) {
 	}
 }
 
+if ( ! function_exists( 'independent_publisher_show_comment_authors_full_name_in_reply_link' ) ):
```

Comment:
@vskjefst Could you also rename this function to `independent_publisher_show_full_name_comment_reply_to`?